pkg/build/api/types.go

```diff
@@ -59,6 +59,10 @@ type BuildSpec struct {
 	// Compute resource requirements to execute the build
 	Resources kapi.ResourceRequirements
 
+	// PostCommit is a build hook executed after the build output image is
+	// committed, before it is pushed to a registry.
+	PostCommit BuildPostCommitSpec
```

Comment:
strictly naming wise as i remember it's trying to outline more the fact that it takes place between the commit and before the push.